run_tests.sh

```diff
@@ -17,8 +17,9 @@ then
     echo "############## Running tests with framework $mlFramework ##############"
     echo "#######################################################################"
 
-    pytest --cov-report=xml --cov=art --cov-append  -q -vv tests/defences/detector/poison/test_spectral_signature_defense.py --mlFramework=$mlFramework --skip_travis=True --durations=0
-    if [[ $? -ne 0 ]]; then exit_code=1; echo "Failed defences/detector/poison/test_spectral_signature_defense.py tests"; fi
+    # TODO: merge defences/detector/poison tests to one line
```

Comment:
Could it be that this TODO has been implemented?

Reply:
Yes it has. Removing on next commit.